scripts/create-kubestellar-demo-env.sh

```diff
@@ -146,26 +206,47 @@ images=("ghcr.io/loft-sh/vcluster:0.16.4"
 
 for image in "${images[@]}"; do
     (
-        docker pull "$image" && kind load docker-image "$image" --name kubeflex
+        docker pull "$image" 
     ) &
 done
+wait
+
+for image in "${images[@]}"; do
+    if [ "$k8s_platform" == "kind" ]; then
+        kind load docker-image "$image" --name kubeflex
+    else
+        k3d image import "$image" --cluster kubeflex
+    fi
+done
 
-echo -e "\nStarting the process to install KubeStellar core: kind-kubeflex..."
-helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
-    --version $kubestellar_version \
-    --set-json='ITSes=[{"name":"its1"}]' \
-    --set-json='WDSes=[{"name":"wds1"}]' \
-    --set-json='verbosity.default=5'
+echo -e "\nStarting the process to install KubeStellar core: $k8s_platform-kubeflex..."
+if [ "$k8s_platform" == "k3d" ]; then
+    helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
+        --version $kubestellar_version \
+        --set-json='ITSes=[{"name":"its1"}]' \
+        --set-json='WDSes=[{"name":"wds1"},{"name":"wds2", "type":"host"}]' \
+        --set-json='verbosity.default=5' \
+        --set "kubeflex-operator.hostContainer=k3d-kubeflex-server-0"
```

Comment:
FYI, you don't need the dquotes on this line. They are harmless here, but killed the previous commit.